jiup/rules/rules.go

```diff
@@ -833,12 +833,11 @@ func init() {
 	Rule("kicad",
 		v.Regexp(
 			"http://kicad-pcb.org/download/windows/",
```

Comment:
Shouldn't it be changed to https://www.kicad.org/download/windows/ ?

Reply:
Thats right, idk where the "-pcb" came from, but it works, so that might be the answer, why i didnt see this error.